wkdehdlr/CoinHelper/app/src/main/java/kr/schoolsharing/coinhelper/base/BaseViewHolder.kt

```diff
@@ -8,8 +8,8 @@ import androidx.databinding.ViewDataBinding
 import androidx.recyclerview.widget.RecyclerView
 
 abstract class BaseViewHolder<B : ViewDataBinding>(
-    @LayoutRes layoutId: Int,
-    parent: ViewGroup,
+    @LayoutRes private val layoutId: Int,
+    private val parent: ViewGroup,
```

Comment:
해당 변수는 접근자 및 프로퍼티가 필요 없어요.
```suggestion
    parent: ViewGroup,
```

Reply:
[5cd2dbc](https://github.com/StudyFork/GoogryAndroidArchitectureStudy/pull/216/commits/5cd2dbc8630cee4d3ef5dd2ecf4917667192df88) 수정했습니다